test/e2e/util.go

```diff
@@ -1032,6 +1032,11 @@ func RunRC(config RCConfig) error {
 		}
 		oldPods = pods
 		oldRunning = running
+
```

Comment:
Just for the record, this function is now 128 lines long - over-ripe for a refactoring, but not necessarily in this PR.